pkg/apis/flow/v1alpha1/xmltojsontransformation_types.go

```diff
@@ -0,0 +1,106 @@
+/*
+Copyright 2022 TriggerMesh Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/triggermesh/triggermesh/pkg/targets/adapter/cloudevents"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/pkg/apis"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/pkg/kmeta"
+)
+
+// +genclient
+// +genreconciler
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// XMLToJSONTransformation is a Knative abstraction that encapsulates the interface by which Knative
+// components express a desire to have a particular image cached.
```

Comment:
The godoc is well written but... how does it relate?

Prefer something generic that can be easily copied for these comments. As an example, _all_ sources use the same generic comment below, this way there is no risk to perform a bad copy-pasta:

https://github.com/triggermesh/triggermesh/blob/67e8d5c90b172de8187ca9ee46f6ef0bb7f30f3a/pkg/apis/sources/v1alpha1/awscloudwatch_types.go#L32-L33